tools/go.mod

```diff
@@ -20,6 +20,8 @@ require (
 	mvdan.cc/gofumpt v0.6.0
 )
 
+require gopkg.in/yaml.v3 v3.0.1
```

Comment:
Please move it to the group above

Reply:
Added to group as requested, and run `task lint` to verify formatting.